std/getopt.d

```diff
@@ -755,8 +766,12 @@ private void getoptImpl(T...)(ref string[] args, ref configuration cfg,
             }
             if (a == "--help" || a == "-h")
             {
-                rslt.helpWanted = true;
-                args = args.remove(i);
+                if (!cfg.noInternalHelp)
+                {
+                    rslt.helpWanted = true;
+                    args = args.remove(i);
+                }
+                else i++;
```

Comment:
I know I am nitpicking, but could you add curly braces on the `else` branch as well? I think that if the `if` clause has braces, then so should the `else` one; I think it brings a nice style.